host-bmc/host_pdr_handler.cpp

```diff
@@ -1060,21 +1043,18 @@ void HostPDRHandler::setHostFirmwareCondition()
                                         size_t respMsgLen) {
         if (response == nullptr || !respMsgLen)
         {
-            std::cerr << "Failed to receive response for "
-                      << "getPLDMVersion command, Host seems to be off \n";
+            lg2::error("Failed to receive response for getPLDMVersion command, Host seems to be off");
             return;
         }
-        std::cout << "Getting the response. PLDM RC = " << std::hex
-                  << std::showbase
-                  << static_cast<uint16_t>(response->payload[0]) << "\n";
+        lg2::error( "Getting the response. PLDM RC = {KEY0}", "KEY0", lg2::hex, (uint16_t)response->payload[0]); 
```

Comment:
same as above, please check all the std::cout traces and make it info rather than error